README.md

```diff
@@ -1,17 +1,19 @@
 # Project Name:-
 
-> Drop dawn popup menu
+> Portfolio popup page 

```

Comment:
- [ ] **Required:** Kindly write your project name without the label. It should be the biggest header on your readme, more like the title of the page.

![image](https://user-images.githubusercontent.com/27709832/163485565-3df9c092-67ef-4727-bc65-b4899ede4a7e.png)

Note that you misnamed your branch. For a future project, always correctly name your branches as this can be confusing for reviewers.